Plugin/Magento/Sales/Model/AdminOrder/CreatePlugin.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Plugin\Magento\Sales\Model\AdminOrder;
+
+use Magento\Sales\Model\AdminOrder\Create;
+use Magento\Backend\Model\Session\Quote as AdminCheckoutSession;
+use Bolt\Boltpay\Helper\Config as ConfigHelper;
+
+class CreatePlugin
+{
+    /**
+     * @var \Bolt\Boltpay\Helper\Config
+     */
+    private $configHelper;
+
+    /** @var AdminCheckoutSession */
+    private $adminCheckoutSession;
+
+    /**
+     * CreatePlugin constructor.
+     * @param ConfigHelper $configHelper
+     * @param AdminCheckoutSession $adminCheckoutSession
+     */
+    public function __construct(
+        ConfigHelper $configHelper,
+        AdminCheckoutSession $adminCheckoutSession
+    )
+    {
+        $this->adminCheckoutSession = $adminCheckoutSession;
+        $this->configHelper = $configHelper;
+    }
+
+    /**
+     * @param Create $subject
+     * @param callable $proceed
+     * @param $data
+     * @return Create
+     */
+    public function aroundImportPostData(Create $subject, callable $proceed, $data)
```

Comment:
It looks like that a _before_ method can be used here. Can you check?
We should avoid _around_ methods unless absolutely necessary.

Reply:
@danac-gs I've updated the code. Can you re-check?